docs/experimental-machine-learning/host-risk-score.md

```diff
@@ -1,105 +1,192 @@
-# Host Risk Score- What is it?
+# Host Risk Score
 
-The Host Risk Score package consists of all the artifacts required to stand up the host risk scoring framework in your environment. This framework leverages transforms and visualizations in Kibana to identify the most suspicious hosts in your environment, based on alert activity on the hosts. 
-To deploy this framework in your environment, follow the steps outlined below.
+Host Risk Score is an experimental feature which assigns risk scores to hosts in a given Kibana space. Risk scores are calculated for each host by transforms on the alerting indices, and they update every hour with the most recent alerting activity. The Host Risk Score [package](https://github.com/elastic/detection-rules/releases) contains all of the required artifacts for setup. Host Risk Score feature is equipped with drilldown dashboards and additional Kibana features such as _the Host Risk Score Card on the Overview Page_ and _the Host Risk Keyword on the Alert Summary Flyout_ for an enhanced experience.
 
-# Detailed steps
+### Notes
+ - **host name collision**: Hosts are identified by the `host.name` field on alerts. There may be some edge cases where different hosts use the same name. [details](#host-name-collision-details) 
 
-#### 1. Unzip the release bundle
+## Setup Instructions
 
-Navigate to the latest GitHub [release](https://github.com/elastic/detection-rules/releases)), with the tag `ML-HostRiskScore-YYYMMDD-N`. From under `Assets`, download the zip file named `ML-HostRiskScore-YYYMMDD-N.zip` and unzip it. New releases may contain updated artifacts. 
+ 1. [Obtain artifacts](#obtain-artifacts) 
+ 2. [Upload scripts](#upload-scripts)
+ 3. [Upload ingest pipeline](#upload-ingest-pipeline)
+ 4. [Upload and start the `pivot` transform](#upload-start-pivot)
+ 5. [Create the Host Risk Score index](#host-risk-index)
+ 6. [Upload and start the `latest` transform](#upload-start-latest)
+ 7. [Import dashboards](#import-dashboards)
+ 8. [Enable Kibana features](#enable-kibana)
 
-#### 2. Modify artifacts to reflect Kibana space
+<h3 id="modify-artifacts">1. Obtain artifacts</h3>
 
-For security reasons, we require that you restrict the Host Risk Score application to a specific Kibana space. In order to do so, run the `ml_hostriskscore_generate_scripts.py` script in the directory that is prefixed with `ML-HostRiskScore` with your Kibana space as the argument.
+The Host Risk Score functionality is space aware for privacy. Downloaded artifacts must be modified with the desired space before they can be used.
 
-Eg:
+ - Download the latest release [bundle](https://github.com/elastic/detection-rules/releases) with the tag `ML-HostRiskScore-YYYMMDD-N`.
+ - Unzip the contents of `ML-HostRiskScore-YYYMMDD-N.zip`.
+ - Run `ml_hostriskscore_generate_scripts.py` script in the unzipped directory with your Kibana space as the argument.
+<div style="margin-left: 40px">   
+<i>e.g. modifying artifacts for default space</i>
```

Comment:
We try to avoid `(e.g.,)` for our non-native English speakers who may not know what it means.
```suggestion
<i>Example of modifying artifacts for default space</i>
```